code/game/mecha/combat/five_stars.dm

```diff
@@ -0,0 +1,22 @@
+/obj/mecha/combat/five_stars
+	desc = "A state of the art tank deployed by the Spinward Stellar Coalition National Guard."
+	name = "\improper Tank"
+	icon = 'icons/mecha/mecha_96x96.dmi'
+	icon_state = "five_stars"
+	armor = list("melee" = 100, "bullet" = 50, "laser" = 35, "energy" = 35, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 100, "acid" = 100)
```

Comment:
Ah yes, a true pinnacle of game balance.

Reply:
I'll prolly tweak this but use explosives tbqh

Reply:
100 melee 50 bullet 100 fire.  It is completely immune to anything that's not a gun or a bomb.

Reply:
Well, stuff with armor piercing (like powercrepes or certain null rod forms) would technically still be able to damage it, but yeah, this is a stupidly high amount of melee armor.